project_wip/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
```

Comment:
Codacy found an issue: [Statement seems to have no effect](https://app.codacy.com/gh/Numigi/odoo-project-addons/pullRequest?prid=6165869)